console/console-init/ui/src/modules/project/components/MessagingConfiguration/MessagingConfiguration.tsx

```diff
@@ -88,10 +88,14 @@ export const MessagingConfiguration: React.FC<IMessagingConfigurationProps> = ({
       <Grid>
         <GridItem span={6}>
           <Form>
-            <FormGroup label="Namespace" isRequired={true} fieldId="name-space">
+            <FormGroup
+              label="Namespace"
+              isRequired={true}
+              fieldId="messaging-config-name-space-dropdowntoggle"
+            >
               <br />
               <DropdownWithToggle
-                id="cas-dropdown-namespace"
+                id="messaging-config-name-space-dropdowntoggle"
```

Comment:
I think you should use dropdwon instead of dropdowntoggle, internally we are using patternfly dropdwon component. 
id="messaging-config-name-space-dropdown"